int16 id
uint8 property
predictposbuffer[] positions
uint8 inactivecount
float32 ave_angle
float32 change_omiga
float32 length
float32 width
float32 height
mapobjcell[] cell
float32 xvrel
float32 yvrel
float32 vrel
float32 xvabs
float32 yvabs
float32 vabs
float32 init_time
float32 change_time
int16 state
float32 theta_toline
int16 stay_count
int16 abs_index
int16 rel_index
int16 direction_y_change
bool direction_y
int16 direction_x_change
bool direction_x
bool theta_line
int16 delta_state
bool update
float32 heading